kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/Utils.java

```diff
@@ -60,6 +60,17 @@ private Utils() {
     return ref;
   }
 
+  public static <T> boolean isNotNull(T[] refList) {
+    if (refList != null) {
+      for (T ref : refList) {
+        if (ref != null) {
+          return true;
+        }
+      }
+    }
+    return false;
+  }
```

Comment:
Nit: Can be replaced by a less complex functional and declarative approach
```suggestion
  public static <T> boolean isNotNull(T... refList) {
    return Optional.ofNullable(refList)
      .map(refs -> Stream.of(refs).allMatch(Objects::nonNull))
      .orElse(false);
  }
```

It would also be good to move this function near the rest of `isNull*`methods

Reply:
Add some Unit Tests to verify and document the behavior of this function:
e.g.
```
  @Test
  @DisplayName("isNotNull, null, should return false")
  public void isNotNullNullTest() {
    // When
    final boolean result = Utils.isNotNull((Object)null);
    // Then
    assertFalse(result);
  }

  @Test
  @DisplayName("isNotNull, some null values, should return false")
  public void isNotNullSomeAreNullTest() {
    // When
    final boolean result = Utils.isNotNull("Not Null", null);
    // Then
    assertFalse(result);
  }

  @Test
  @DisplayName("isNotNull, no null values, should return true")
  public void isNotNullNoneAreNullTest() {
    // When
    final boolean result = Utils.isNotNull("Not Null", "Not Null either");
    // Then
    assertTrue(result);
  }
```